### PR TITLE
Greg training kubernetes

### PR DESCRIPTION
**Pull request by GregdTd**

### CODE REVIEW

File: kubernetes/greg-10-08-2022/README.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+
+Running `k logs greg-pod` which gives `Running on http://0.0.0.0:8080` so I've changed the `targetPort` of the service from `9376` to `8080`. Also I've removed the `containerPort` of the pod.yaml as it seems unused.
+
+### Make your API publicly accessible
+
+Write an ingress that makes your service accessible to the world.
+
+**Validation**
+
+`curl <your-ingress-address>` returns hello world from your pod.
```
Comment (NaitoKenzo) [inline kubernetes/greg-10-08-2022/README.md:87]: From which pod, do you know?

File: kubernetes/greg-10-08-2022/README.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+Annotations:  alb.ingress.kubernetes.io/scheme: internet-facing
+              alb.ingress.kubernetes.io/target-type: ip
+Events:
+  Type    Reason                  Age   From     Message
+  ----    ------                  ----  ----     -------
+  Normal  SuccessfullyReconciled  3s    ingress  Successfully reconciled
+
+```
+
+Wouhouuu validation is working, running `curl k8s-default-gregingr-ce2d40507d-1389164601.eu-north-1.elb.amazonaws.com` return "Hello World"
```
Comment (NaitoKenzo) [inline kubernetes/greg-10-08-2022/README.md:173]: This journal is rather fun to read!

File: kubernetes/greg-10-08-2022/README.md [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+
+`curl <your-ingress-address>/author` return { "author": "<your-name>" }
+
+### What I did/learn
+
+I followed two docs: [1](https://kubernetes.io/docs/concepts/configuration/configmap/) and [2](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/)
+
+However when getting the author route I got a 404.
+I checked successfully that the env variable was present in pod using `k exec -it greg-pod -- bash` and `echo $AUTHOR`
+I suspect a wrong docker image.
```
Comment (NaitoKenzo) [inline kubernetes/greg-10-08-2022/README.md:189]: What was wrong with the image?

File: kubernetes/greg-10-08-2022/pod.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+      #   - name: AUTHOR
+      #     valueFrom:
+      #       configMapKeyRef:
+      #         name: greg-configmap
+      #         key: author
+      envFrom:
+        - configMapRef:
+            name: greg-configmap
+      # Taking random values from kubernetes docs.
+      resources:
```
Comment (NaitoKenzo) [inline kubernetes/greg-10-08-2022/pod.yaml:21]: This is specified in the deployment so not necessary here.

File: kubernetes/greg-10-08-2022/replicaset.yaml [hunk 5]
```diff
@@ -0,0 +1,24 @@
+apiVersion: apps/v1
+kind: ReplicaSet
```
Comment (NaitoKenzo) [inline kubernetes/greg-10-08-2022/replicaset.yaml:2]: The ReplicaSet can be described in the deployment

Comment (NaitoKenzo) [reply kubernetes/greg-10-08-2022/replicaset.yaml:2]: I wonder how many pods were running. Do you know? I'd expect 5

File: kubernetes/greg-10-08-2022/README.md (hunk 3 above)
Comment (GregdTd) [reply kubernetes/greg-10-08-2022/README.md:189]: There were no route /author

**Pull request merged**